.github/workflows/pr-test.yml

```diff
@@ -11,6 +11,21 @@ jobs:
     with:
       python-version: "3.9"
 
+  danger-check:
+    runs-on: [ self-hosted, public, linux, x64 ]
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9  # v3
+      - name: Install Node.js
+        uses: actions/setup-node@e33196f7422957bea03ed53f6fbb155025ffc7b8  # v3
+        with:
+          node-version: "16"
+      - name: Install and run DangerJS
+        env:
+          GITHUB_TOKEN: ${{ secrets.DANGER_GITHUB_TOKEN }}
+        run: |
+          npm install -g danger
```

Comment:
since you install danger here directly, we probably don't need the `package.json`